<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit</class>
 <widget class="QWidget" name="edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_New"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>127</y>
     <width>781</width>
     <height>501</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_horizontalFlip">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>280</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizontal Flip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_verticalFlip">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>230</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical Flip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>180</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>130</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>200</width>
     <height>501</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="roomsPage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>396</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Rooms</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Rectangle Room</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
       <widget class="QToolButton" name="btn_squareRoom">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Images/square.png</normaloff>Images/square.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
       <widget class="QToolButton" name="btn_rectangleRoom">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Images/rectangle.png</normaloff>Images/rectangle.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignTop">
       <widget class="QLabel" name="lbl_squareRoom">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Square Room</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
       <widget class="QToolButton" name="btn_door">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Images/door.png</normaloff>Images/door.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
       <widget class="QLabel" name="label_15">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Door</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="furniturePage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>396</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Furniture</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>191</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
       <widget class="QToolButton" name="btn_Chest">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Images/chest.png</normaloff>Images/chest.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Chest</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
       <widget class="QToolButton" name="btn_Chair">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Images/chair.png</normaloff>Images/chair.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Table</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
       <widget class="QToolButton" name="btn_Table">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Images/table.png</normaloff>Images/table.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Chair</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="flooringPage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>396</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Flooring</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>191</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
      </property>
      <item row="2" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
       <widget class="QToolButton" name="btn_friezeCut">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Images/frieze.png</normaloff>Images/frieze.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
       <widget class="QToolButton" name="btn_hardFloor">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Images/hardfloor.png</normaloff>Images/hardfloor.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Hard Floor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
       <widget class="QToolButton" name="btn_loopPile">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Images/looppile.png</normaloff>Images/looppile.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
       <widget class="QToolButton" name="btn_cutPile">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Images/cutpile.png</normaloff>Images/cutpile.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Loop Pile</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Cut Pile</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Frieze-Cut</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="action_About">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSaveAs"/>
   </property>
   <property name="text">
    <string>&amp;Save As</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
